packages/field/stories.tsx

```diff
@@ -11,32 +11,29 @@ import { storiesOf } from '@storybook/react';
 import { withKnobs, text } from '@storybook/addon-knobs';
 
 import { FieldContainer, useField } from './src';
+import { generateId } from '@zendeskgarden/container-utilities/src';
```

Comment:
Why does this reach into the `src`? We should be ok with the standard `dist` folder

Reply:
Ah yes! I'll update this to pull from standard `dist` folder. Thanks!